src/plone/restapi/services/vocabularies/configure.zcml

```diff
@@ -17,7 +17,7 @@
       accept="application/json"
       factory=".get.VocabulariesGet"
       for="Products.CMFCore.interfaces.IContentish"
-      permission="plone.restapi.vocabularies"
+      permission="zope2.View"
```

Comment:
See below.
```suggestion
      permission="zope2.Public"
```

Reply:
keep as is - lets follow the old way at https://github.com/plone/plone.app.content/blob/2b5445f9c34c5a897471a3fe2823d12527c6f95a/plone/app/content/browser/configure.zcml#L166